Gemfile

```diff
@@ -10,6 +10,11 @@ gem "rails", rails_version == "main" ? {git: "https://github.com/rails/rails", r
 
 gem "rspec-rails", "~> 5"
 
+# Utilize to test interactions
+gem "cuprite", "~> 0.13"
+gem "puma", "~> 5.4"
+gem "selenium-webdriver"
+
```

Comment:
I'm wondering if there's a way we can avoid adding these as gem dependencies. Folks may want to use other browser drivers, for example. (We can't use cuprite due to incompatibilities in `axe-core`, for example)

```suggestion
```

Reply:
(we could still add them as dev dependencies for the sake of testing this feature in CI)

Reply:
My thinking is that we'd have something like:

`visit(rendered_component_path(MyComponent.new, layout: "special"))`

Reply:
My thinking is that we'd have something like:

`visit(rendered_component_path(MyComponent.new, layout: "special"))`, that way we'd let folks handle `visit` however they please, limiting ViewComponent's responsibility to providing the route to visit.

Reply:
I think the best we could do from just a quick glance is to put them as dev dependencies. You are right - people may have different preferences.